### ISSUE TITLE
Add Automated Accessibility Testing

### ISSUE THREAD

**Issue by martypowell** [Baltimore-County-Government/dotgov-components#31]
Checkout: https://www.npmjs.com/package/cypress-axe

I think this would only work once we started developing the homepage. The intial test would load the homepage, and run the test.

```js
it('Has no detectable a11y violations on load', () => {
  // Test the page at initial load
  cy.checkA11y()
})
```

fyi @danfox01
